objcts.js

```diff
@@ -0,0 +1,9 @@
+for(i=1; i<=15;i++){
+let k = i;
+let j = k*k;
+let numbers ={
+[k] : [j],
+}
+for(i in numbers){
```

Comment:
You won't need this but still if you ever use the `for...in` loop, don't forget to declare the variable first i.e
It would look like
```js
for(var i in numbers)
```